cmake-scripts/Findminiupnpc.cmake

```diff
@@ -1,3 +1,16 @@
+if(VCPKG_TOOLCHAIN)
+  if(TARGET miniupnpc::miniupnpc)
+    return()
+  endif()
+  find_package(miniupnpc CONFIG)
+  if(TARGET miniupnpc::miniupnpc)
+    MESSAGE(STATUS "Found miniupnp in vcpkg")
+    set(MINIUPNPC_LIBRARY miniupnpc::miniupnpc)
+    SET(MINIUPNPC_FOUND TRUE)
+    set(MINIUPNPC_INCLUDE_DIR)
+    return()
+  endif()
+endif(VCPKG_TOOLCHAIN)
```

Comment:
This has uneven indentation compared to rest of the file, and also strange that `set` is sometimes in capital letters, sometimes not?

Reply:
Perhaps we should invest in setting up [cmake-format](https://github.com/cheshirekow/cmake_format)?
For now, this shim matches the style of the other find_package(CONFIG) shims

I'll make a seperate PR to master for cmake-format

Reply:
Opened  #710